packages/gatsby-source-shopify/README.md

```diff
@@ -127,6 +132,26 @@ To create a production build, use gatsby build
 
 Now follow the second link to explore your Shopify data!
 
+<div id="priority-builds"></div>
+
+### Priority builds
+
+Because of the limiations of the Shopify Bulk API we have included logic in this plugin to determine which builds are high priority for a given Shopify site. This allows us to pause deploy preview builds while production builds are running while using the same Shopify App. The following logic determines whether a build is priority or not:
```

Comment:
@byronlanehill shouldn't we say "cancel preview builds" rather than "pause preview builds" here? Or do we actually have logic to pause preview builds and retry later on?

Reply:
@veryspry We do have logic that pauses the build assuming it is not prioritized. The only time a build would fail is if it is prioritized and is canceled anyways, which would indicate that there are multiple prioritized builds running concurrently.